conda_build/variants.py

```diff
@@ -31,6 +31,7 @@
     'ignore_version': [],
     'ignore_build_only_deps': ['python'],
     'extend_keys': ['pin_run_as_build', 'ignore_version', 'ignore_build_only_deps'],
+    'cran_mirror': "https://cran.r-project.org",
```

Comment:
Should we be adding this to `conda-forge-pinning`?

Reply:
IMHO you are better off using MRAN to take advantage of snapshotting.

The R world is better organised than the Python world and they make a big effort to get things working for major releases. After that date, the chances of breakages are much higher as more risky versions will get released.

Reply:
Am not an R user/maintainer either. So would let R users/maintainers in conda-forge make this decision.

Was more asking if we should make this explicit in `conda-forge-pinning` to make it clear to users what we are using as a mirror. Sounds like it is only used for `conda skeleton cran`, in which case there is no real point in setting it. Is that correct?

Reply:
It is used in the skeleton, and it is also used when building recipes produced by the skeleton.  There is some point in setting it.